addons/point_of_sale/static/src/app/screens/scale_screen/scale_screen.js

```diff
@@ -15,21 +15,22 @@ export class ScaleScreen extends Component {
     setup() {
         this.pos = usePos();
         this.hardwareProxy = useService("hardware_proxy");
-        this.state = useState({ weight: 0 });
+        this.state = useState({ weight: 0, tare: 0, tareLoading: false });
+        this.pricePerUom = 0;
```

Comment:
```suggestion
        this.pricePerUom = product.get_price(this._activePricelist, 1);
```

In fact it's a constant, so you can just assign it like this.